.travis.yml

```diff
@@ -0,0 +1,35 @@
+addons:
+  apt:
+    packages:
+      - nodejs
+language: python
+sudo: false
+python:
+  - 3.5
+env:
+  - TOX_ENV=django111
+  - TOX_ENV=django20
+  - TOX_ENV=django21
+  - TOX_ENV=django22
+matrix:
+  include:
+    - python: 3.5
+      env: TOX_ENV=csslint
+    - python: 3.5
+      env: TOX_ENV=eslint
+    - python: 3.5
+      env: TOX_ENV=pycodestyle
+    - python: 3.5
+      env: TOX_ENV=pylint
+before_install:
+  - "pip install -U pip"
+  - export BOTO_CONFIG=/dev/null
+cache:
+  directories:
+    - $HOME/.cache/pip
+install:
+  - "make requirements"
+  - "pip install coveralls"
```

Comment:
We've switched to codecov in most places; had a lot of problems with coveralls reliability a while back.

Reply:
Ah, I cribbed much of this from the ORA2 xblock, which still uses coveralls.

Can we punt for now?

Regardless, good to know for the future.

Reply:
Done.